src/tutorials/first_game_br.md

```diff
@@ -10,116 +10,162 @@ tags:
 
 ## Overview
 
-The Battle Royale framework features a game where the player must skydive to a starting location, the play area shrinks over time, and the last player standing wins. It will also include random weapon spawns using a loot table.
+The Battle Royale framework features starting logic for game where the player must skydive to a starting location, the play area shrinks over time, and the last player standing wins. It also includes random weapon spawns using a loot table.
 
-## Create a Framework Game
-
-To get started, use the **CREATE** menu to make a new project
+## Creating a Framework Game
 
 ### Open the **NEW PROJECT** Menu
 
-![Create New](../img/NewFrameworks/NewFrameworks_NewProject.png){: .center loading="lazy" }
-
 1. With Core open, click the **CREATE** tab in the top menu bar.
 2. Click the **NEW PROJECT** tab at the top of the **CREATE** menu.
 
-### Select the **Battle Royale** Framework
+![!Create New](../img/NewFrameworks/NewFrameworks_NewProject.png){: .center loading="lazy" }
 
-<!--- TODO image of BR framework choice --->
+### Select the **Battle Royale** Framework
 
 1. In the **GAMEPLAY FRAMEWORKS** section, select **Battle Royale**.
 2. In the right side panel, name your project.
 3. Click **CREATE**.
 
-## Modify the Airplane Template
+![!Battle Royale](../img/FrameworksBR/FrameworksBR_Selection.png){: .center loading="lazy" }
+
+## Testing the Framework
+
+### Preview in Multiplayer
+
+The framework requires a minimum of two players to play, so you will need to test in **Multiplayer Preview Mode**.
+
+![TwoPlayers](../img/MyFirstMultiplayer/TwoPlayers.png){: .center loading="lazy" }
+
+1. Click ![Multiplayer Preview Mode](../img/EditorManual/icons/Icon_MultiplayerTest.png) to switch the Preview Mode to Multiplayer.
+{: .image-inline-text .image-background }
+2. Select the **2 Players** option for the number of players.
+3. Press ![Play](../img/EditorManual/icons/Icon_Play.png) to start the preview. This will open a separate game window for each player.
+{: .image-inline-text .image-background }
 
-The game starts with the players inside of a paper airplane. You can modify the appearance of the airplane by updating the **Helper Plane** template.
+### Test the Features
 
-<!--- TODO add image of default airplane --->
+Test out the following features in the framework:
+
+- Players spawn on the ground.
+- After a couple seconds, the players will be moved inside of a Paper Plane.
+- Players can drop from the plane using ++space++.
+- Players can activate the parachute while dropping using ++space++.
+- Players will be dealt damage if outside the Play Zone.
+- The Play Zone will shrink over time.
+- Loot Spawners around the map contain one of three weapons:
+    - Basic Pistol
+    - Basic Shotgun
+    - Basic Rifle
+- The round ends once there is one player remaining alive.
+
+!!! note "The video below has modified game settings to speed up the testing, which will be covered in this tutorial."
```

Comment:
Try not to reference content spatially (the "below" here). Just note that you'll be changing settings to speed up testing in the tutorial. Or retake the video with just the base project.